name.py

```diff
@@ -0,0 +1,3 @@
+name = input("escribe tu nombre bro:")
```

Comment:
Oye, puedes poner de mejor forma esta string
```suggestion
name = input("escribe tu nombre bro: ")
```